### ISSUE TITLE
[feat] Statistics view showing all statistics of all devices in single view

### ISSUE THREAD

**Issue by alexruffell** [zwave-js/zwavejs2mqtt#2552]
**Is your feature request related to a problem? Please describe.**
I have 90 or so devices. I find the statistics to be very useful but very hard to take advantage of as to find what devices are having issues I have to open the statistics for each device. Two clicks per device plus the scrolling means nearly 200 mouse operations to view all devices.

**Describe the solution you'd like**
A single view in a tabular format with each column showing device statistics would make it super quick to identify what devices are having issues. It would be nice for the view to also provide information such as when the statistics shown started (I assume last restart of zwavejs2mqtt but I don't know when that was with good accuracy without further hunting). Tool tip, or other, explaining a bit more what each tooltip means, and some way to raise awareness of what is bad would be nice. High message numbers may be bad if collected in a short period of time as they may represent excessive traffic, but may be OK if the collection time is extended. Showing what devices cause the majority of traffic would be great (percentage of total related messages) 

**Describe alternatives you've considered**
Export to CSV file

**Additional context**
I have always had issues with excessive traffic on my mesh network. Issues were usually related to very chatty devices but I never had proof on which ones specifically or good ways to see the effects of my changes when trying to tone them down.

**Comment by robertsLando** [comment]
Hi @alexruffell ! Thanks for your issue. Looks interesting and super useful. I think that a general indication about how the device is performing could be implemented on zwavejs side too. Something that reports the general status of a node depending on how stats change over the time.

@AlCalzone ideas?

**Comment by AlCalzone** [comment]
AFAIK z2m already has access to all the information necessary to generate such a view. Surely zwave-js could be extended give some hints, but the raw data is there as a first step.

**Comment by robertsLando** [comment]
Yep I already have all the information needed for it, just wondering if it could make sense to add to statistics updated event something that tells us an health status of that node based on them instead of implementing that logic on my side

**Comment by AlCalzone** [comment]
I have a couple of plans in this regard, will take some time though.
